packages/components/cascader-panel/src/node.vue

```diff
@@ -167,10 +167,10 @@ export default defineComponent({
     }
 
```

Comment:
Solutions for radio.
1、no need to load data
2, if the data has been loaded, then you should doExpand (otherwise the right panel data and the left side are not associated)

```js
const handleRadioCheck = (checked: boolean) => {
  doCheck(checked)
  if (props.node.loaded) {
    doExpand()
  }
}
```